README.md

```diff
@@ -0,0 +1,83 @@
+# PyVista Tutorial
+
+## Abstract
+
+This tutorial will introduce the 3D visualization using PyVista (https://github.com/pyvista/pyvista).
+PyVista is 3D plotting and mesh analysis through a streamlined interface for the Visualization Toolkit (VTK).
+PyVista uses VTK (http://www.vtk.org) under the covers and provides a very Python-friendly API.
+
+At the end of the tutorial users will be able to
+
+- Use PyVista effectively to plotting 3D visualizations
+- Understand the basic class structure of PyVista.
+
+## Keywords
+
+[visualization](https://github.com/topics/visualization)
+[meshviewer](https://github.com/topics/meshviewer)
+[vtk](https://github.com/topics/vtk)
+[3d](https://github.com/topics/3d)
+
+## Other Information and Files
+
+### Tutorial Description.
+Same with Abstract.
+
+### Tutorial Outline.
```

Comment:
Most of the "train"s in this section should probably be "training", but there might be an even better word. I'm not sure.

Reply:
"exercise" may prefer.
> A more detailed outline of the tutorial content, including the duration of each part and exercise sessions. Please include a description of how you plan to make the tutorial hands-on.